apps/webbsite/src/components/sections/BlogSection.tsx

```diff
@@ -0,0 +1,91 @@
+import { useMemo, useState } from 'react';
+import { PostsOrVideos } from '../../libs/notion';
+import { BlogCard, Heading3 } from '../../components';
+import { Button } from '@webb-tools/webb-ui-components';
+import Link from 'next/link';
+
+type BlogSectionProps = {
+  type: 'post' | 'video';
+  items: PostsOrVideos[];
+  showAllItems?: boolean;
+};
+
+export const BlogSection = (data: BlogSectionProps) => {
+  const { type, items, showAllItems = false } = data;
```

Comment:
Suggestions:

1. Destructure the `data` parameter directly in the function signature instead of inside the function body. This will make the code easier to read and understand.

2. Since `tagsArrays` is only used to compute the `tags` array using the `reduce` function, you can skip the intermediate `tagsArrays` variable and chain the `reduce` function directly on allItems.

3. You can simplify the logic in the `tags` and `filteredItems` memoized values by using the ternary operator and eliminating the else branch.

4. Instead of hard-coding the `BlogCard` component's properties, you can destructure the metadata object directly in the function argument.


```suggestion
export const BlogSection = ({ type, items, showAllItems = false }: BlogSectionProps) => {
```